lstchain/irf/hdu_table.py

```diff
@@ -8,15 +8,20 @@
 from astropy.time import Time
 
 #from lstchain.reco.utils import get_effective_time
+from lstchain.version import get_version
 
 __all__ = ["create_obs_hdu_index", "create_event_list"]
 
 log = logging.getLogger(__name__)
 
 DEFAULT_HEADER = fits.Header()
+DEFAULT_HEADER["CREATOR"] = f"lstchain v{get_version(pep440=True)}"
```

Comment:
just use `lstchain.__version__`, then this code does not need to be changed when / if we also switch to using `setuptools_scm` instead of this version.py

Reply:
But then is it ok to have the githash as well?

Reply:
Yes! we should have the git hash in this in case anyone is producing files without a release version

Reply:
Understood! I will correct it accordingly